frontend/src/global_styles/content/work_packages/inplace_editing/_display_fields.sass

```diff
@@ -150,7 +157,7 @@ display-field
   .display-field--scheduling-icon
     position: absolute
     left: 4px
-    top: 4px
+    top: 6px
```

Comment:
You also have to modify the [rule for the placeholder](https://github.com/opf/openproject/blob/release/15.4/frontend/src/global_styles/content/work_packages/inplace_editing/_display_fields.sass#L158), currently it is to high:
<img width="173" alt="Bildschirmfoto 2025-03-12 um 15 02 33" src="https://github.com/user-attachments/assets/c586582f-dc11-4f3a-88ee-82da381306c7" />